README.md

```diff
@@ -38,11 +38,11 @@ The following steps will build both the patched and unpatched binaries in `build
 The following packages are required for building the challenges on Linux:
 
 ```
-libc6-dev
-libc6-dev-i386
-gcc-multilib
-g++-multilib
-clang-3.8 (or higher)
+libc6-dev libc6-dev-i386 gcc-multilib g++-multilib clang clang-3.8 # (or higher)
```

Comment:
Can we just list the `clang` package? It will bring in the `clang-3.8` package automatically, while also creating the required `/usr/bin/clang` symbolic link.

Reply:
Yep, that appears to work fine.